the-blue-alliance-ios/View Controllers/Events/Match/MatchesViewController.swift

```diff
@@ -49,7 +49,7 @@ class MatchesViewController: TBATableViewController {
 
     private func setupFetchRequest(_ request: NSFetchRequest<Match>) {
         if let team = team {
-            request.predicate = NSPredicate(format: "event == %@ AND (ANY redAlliance == %@ OR ANY blueAlliance == %@)", event, team, team)
+            request.predicate = NSPredicate(format: "event == %@ AND (ANY alliances.teamsJoined CONTAINS[c] %@)", event, team.key!)
```

Comment:
This incorrectly matches teams if they have a suffix - ex: a teamsJoined of `frc7332B,frc3333,frc1111` will match for `frc7332`, since we're not matching the end